resources/lang/de/Config.properties

```diff
@@ -350,7 +350,6 @@ addressFieldEnAddress3 = Adresse 3
 addressFieldEnAddress4 = Adresse 4
 addressFieldEnBirthday = Geburtsdatum
 addressFieldEnContactPerson = Ansprechpartner
-addressFieldEnName1 = Firma
```

Comment:
Mit _addressField**En**Name1_ löscht du das falsche Feld, _adressFieldName1_ ist das Duplikat

Reply:
Oh ja.. Da hab eich mit beim löschen in Git vertan..  :-/

Jetzt ist es richtig.